.github/workflows/pypi-publish-kensu-pyspark-on-release.yml

```diff
@@ -1,9 +1,8 @@
 name: Publish kensu-pyspark minimal Python 🐍 Agent
 
 on:
-  push:
-    branches: [ release/py3.5-kensu-pyspark ]
-  workflow_dispatch:
+  release:
+    types: [created]
```

Comment:
now kensu-pyspark release yml is same as original, except extra DELIVERY_ARTIFACT_NAME below